examples/autotagger/src/locations/Home.tsx

```diff
@@ -5,11 +5,6 @@ import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
 
 const Home = () => {
   const sdk = useSDK<HomeAppSDK>();
-  /*
```

Comment:
nit: maybe also get rid of the commented out `useCMA` on line 4 like you did in the other files?